Source/Scene/PointPrimitive.js

```diff
@@ -0,0 +1,488 @@
+/*global define*/
+define([
+        '../Core/Cartesian2',
+        '../Core/Cartesian3',
+        '../Core/Cartesian4',
+        '../Core/Color',
+        '../Core/createGuid',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/DeveloperError',
+        '../Core/Matrix4',
+        '../Core/NearFarScalar',
+        './SceneMode',
+        './SceneTransforms'
+    ], function(
+        Cartesian2,
+        Cartesian3,
+        Cartesian4,
+        Color,
+        createGuid,
+        defaultValue,
+        defined,
+        defineProperties,
+        DeveloperError,
+        Matrix4,
+        NearFarScalar,
+        SceneMode,
+        SceneTransforms) {
+    "use strict";
+
+    /**
+     * A glPoint positioned in the 3D scene, that is created
```

Comment:
You can just leave it if you want, but as we move toward the Entity API, I am planning on going light on the primitive doc, and then heavier on the entity doc.

Billboard/billboard collection was the first thing I wrote in the original Cesium after the globe and I'm sure I did overkill on the doc.